tests/tests_pytorch/conftest.py

```diff
@@ -190,6 +190,21 @@ def tmpdir_server(tmpdir):
         server.shutdown()
 
 
+@pytest.fixture
+def clean_import():
+    import sys
+
+    new_sys_modules, old_sys_modules = sys.modules.copy(), sys.modules
+    pl_keys = list(
+        key for key in new_sys_modules.keys() if key.startswith("pytorch_lightning") or key.startswith("lightning")
+    )
+    for pl_key in pl_keys:
+        new_sys_modules.pop(pl_key, None)
```

Comment:
Aren't the keys guaranteed to be in the dictionary? since you just extracted them from it.

Reply:
Well... Yes, but I have an experience that one cannot be too careful when dealing with `sys.modules` and _weird_ things happen with that dictionary. That's why I'm being extra careful in there.

Reply:
Weird things shouldnt happen during our tests...

Reply:
Alright, done. :rocket: